docs/SETUP.md

```diff
@@ -125,6 +125,12 @@ The CLI provides example values, but I'll also put the configuration I'm using (
 **That's all you need for a functioning ILP Kit!** To start your ILP Kit, run:
 
 ```
+$ npm install; npm rebuild; npm start
```

Comment:
we should mention that if you run this interactively in an ssh session, the process will die when you log out